.github/workflows/non-target-specific.yml

```diff
@@ -0,0 +1,32 @@
+name: Non-target-specific
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      all:
+        type: boolean
+  pull_request:
+    branches-ignore:
+    - 'nightly'
+
+env:
+  # 2020.11
+  vcpkgGitRef: 0bf3923f9fab4001c00f0f429682a0853b5749e0
+
+concurrency:
+  group: only-non-target-specific-${{ github.ref }}-${{ github.event_path }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  # Run the unit tests.
+  junit:
+    if: ${{ inputs.all || github.event.pull_request.draft }}
+    uses: ./.github/workflows/unit-tests.yml
+    with:
+      all-platforms: ${{ !github.event.pull_request.draft  }}
+
+  # Run the serialization tests
+  serialization:
```

Comment:
I think the serialization test should be grouped together with the C tests. The serialization syntax is only supported by the C target, and the test itself work very similar to our standard integration tests.

Reply:
Hm, there are serialization tests for other targets as well. I think this is the right place to have it.